test/e2e/tests/request-queuing/multi-dapp-sendTx-revokePermission.spec.js

```diff
@@ -95,26 +86,37 @@ describe('Request Queuing for Multiple Dapps and Txs on different networks revok
 
         // Dapp 1 send tx
         await driver.switchToWindowWithUrl(DAPP_URL);
-        await driver.delay(largeDelayMs);
+        await driver.findElement({
+          css: '[id="chainId"]',
+          text: '0x1',
+        });
         await driver.clickElement('#sendButton');
 
         await driver.waitUntilXWindowHandles(4);
         await driver.delay(3000);
```

Comment:
this cannot be removed, otherwise the transaction is still using the old  network (Mainnet) so we need to give it some time to switch